datafusion/core/src/physical_plan/limit.rs

```diff
@@ -365,30 +365,17 @@ impl ExecutionPlan for LocalLimitExec {
     }
 }
 
-/// Truncate a RecordBatch to maximum of n rows
-pub fn truncate_batch(batch: &RecordBatch, n: usize) -> RecordBatch {
-    let limited_columns: Vec<ArrayRef> = (0..batch.num_columns())
-        .map(|i| limit(batch.column(i), n))
-        .collect();
-
-    RecordBatch::try_new(batch.schema(), limited_columns).unwrap()
-}
-
 /// A Limit stream skips `skip` rows, and then fetch up to `fetch` rows.
 struct LimitStream {
-    /// The number of rows to skip
+    /// The remaining number of rows to skip
     skip: usize,
```

Comment:
The meaning of`skip` and `fetch` has changed, we should rename them.

Reply:
Oh, I see this was not yet done -- @HaoYang670  can you please do this as a follow on PR? If you don't have time I will do so myself

Reply:
Sure, I'll file a follow-up.